components/CoinListHeader.js

```diff
@@ -27,7 +22,7 @@ export default class CoinListHeader extends Component {
         {this.props.buttons.map((button, index) =>
           (<CoinListHeaderButton
             index={index}
-            onPress={this.handleHeaderButtonPress.bind(this)}
```

Comment:
Yea I think you only need to `.bind(this)` if you're passing that function somewhere else

Reply:
👍